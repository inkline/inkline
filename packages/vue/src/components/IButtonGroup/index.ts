export { default as IButtonGroup } from '@inkline/inkline/components/IButtonGroup/IButtonGroup.vue';
export * from '@inkline/inkline/components/IButtonGroup/types';
