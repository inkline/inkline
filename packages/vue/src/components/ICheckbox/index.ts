export { default as ICheckbox } from '@inkline/inkline/components/ICheckbox/ICheckbox.vue';
