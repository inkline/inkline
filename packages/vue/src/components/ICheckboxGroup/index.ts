export { default as ICheckboxGroup } from '@inkline/inkline/components/ICheckboxGroup/ICheckboxGroup.vue';
export * from '@inkline/inkline/components/ICheckboxGroup/types';
