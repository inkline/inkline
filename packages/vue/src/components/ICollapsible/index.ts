export { default as ICollapsible } from '@inkline/inkline/components/ICollapsible/ICollapsible.vue';
export * from '@inkline/inkline/components/ICollapsible/types';
