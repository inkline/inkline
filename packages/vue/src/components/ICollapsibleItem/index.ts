export { default as ICollapsibleItem } from '@inkline/inkline/components/ICollapsibleItem/ICollapsibleItem.vue';
