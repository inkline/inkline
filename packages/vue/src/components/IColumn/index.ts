export { default as IColumn } from '@inkline/inkline/components/IColumn/IColumn.vue';
