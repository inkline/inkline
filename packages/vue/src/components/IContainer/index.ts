export { default as IContainer } from '@inkline/inkline/components/IContainer/IContainer.vue';
