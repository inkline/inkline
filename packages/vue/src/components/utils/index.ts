export * from '@inkline/inkline/components/utils/ICheckableButtonGroup';
export * from '@inkline/inkline/components/utils/IRenderResolver';
