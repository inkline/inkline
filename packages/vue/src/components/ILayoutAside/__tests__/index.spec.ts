import { render } from '@testing-library/vue';
import { ILayoutAside } from '@inkline/inkline/components';
import { InklineKey } from '@inkline/inkline/constants';
import { createInkline } from '@inkline/inkline/__tests__/utils';

describe('Components', () => {
    describe('ILayoutAside', () => {
        const props = {};

        it('should be named correctly', () => {
            expect(ILayoutAside.name).toEqual('ILayoutAside');
        });

        it('should render correctly', () => {
            const wrapper = render(ILayoutAside, {
                props,
                global: {
                    provide: {
                        [InklineKey as symbol]: createInkline()
                    }
                }
            });

            expect(wrapper.html()).toMatchSnapshot();
        });
    });
});
