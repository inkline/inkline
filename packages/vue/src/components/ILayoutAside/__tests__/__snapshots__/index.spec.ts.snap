// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Components > ILayoutAside > should render correctly 1`] = `
"<aside class="layout-aside">
  <div class="layout-aside-children">
    <!-- @slot default Slot for default layout aside content -->
  </div>
</aside>"
`;
